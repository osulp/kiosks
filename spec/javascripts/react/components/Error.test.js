import {
  mapStateToProps,
  mapDispatchToProps
} from "../../../../app/javascript/react/components/Error"
import * as factories from "../.factories"

describe("Error", () => {
  it("maps state to props", () => {
    expect(
      mapStateToProps({ kiosk: { errors: [factories.error] } })
    ).toMatchSnapshot()
  })

  it("maps dispatch to props", () => {
    expect(mapDispatchToProps(jest.fn())).toMatchSnapshot()
  })
})
