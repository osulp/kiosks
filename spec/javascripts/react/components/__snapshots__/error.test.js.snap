exports[`Error maps dispatch to props 1`] = `
Object {
  "addError": [Function],
  "fetchSlides": [Function],
  "fetchedSlides": [Function],
  "fetchingSlides": [Function],
  "scrollToSlide": [Function],
  "setKiosk": [Function],
  "setModalRootComponent": [Function],
  "setModalVisibility": [Function],
  "setSlides": [Function],
}
`;

exports[`Error maps state to props 1`] = `
Object {
  "errors": Array [
    Object {
      "code": "500",
      "message": "Error",
    },
  ],
}
`;
